css/global.css

```diff
@@ -605,6 +639,28 @@ header nav li:last-child {
 Global Header Styles
 ===================================
 */
+
```

Comment:
**general header + hero styles moved to global.css**

All that needs to be edited per page.css is the 

- background image per page
- header text layouts in each page.css 
- mobile queries (changing from background img to img)

take a look at other pages for reference 

![Screenshot 2024-01-09 174509](https://github.com/indigenousfriends/itlct-website-development/assets/98284615/536b56d9-f1aa-440e-8ab6-14dfd605b7da)

![Screenshot 2024-01-09 174726](https://github.com/indigenousfriends/itlct-website-development/assets/98284615/9b8ddbc2-6e77-4582-bdae-3ef800045758)

Reply:
This is perfect, really clean!

Reply:
thank you!